src/content/structured/components/popover-menu/accessibility.mdx

```diff
@@ -0,0 +1,48 @@
+---
+path: "/components/popover-menu/accessibility"
+
+date: "2023-03-31"
+
+title: "Popover menu"
+
+status: "BETA"
+
+subTitle: "Use popover menus to place actions inside a menu that overlays other page content when opened."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/components/popover-menu"
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/popover-menu" },
+    { title: "Code", path: "/components/popover-menu/code" },
+    { title: "Accessibility", path: "/components/popover-menu/accessibility" },
+  ]
+---
+
+## Easy to use for everyone
+
+Popover menus use the role ‘menu’ and the items within use ‘menuitem’ and ‘menuitemcheckbox’. This enables the menu to be opened using usual keyboard commands. Clicking on a ‘menuitem’ close the menu after triggering the ‘menuitem’ action. Popover menu groups use the role ‘group’.
+
+Popover menus appear overlaid on other content. When opening a submenu, focus is applied to the whole sub-menu and then the items are navigated between using the arrow keys. When focus is moved on, the popover menu closes automatically. Using the ‘Esc’ key will also close the popover menu.
+
+Disabled menu items can still be focussed upon so that the structure of the menu is understandable, even if the menu item can’t be triggered at that time.
+
+## For Assistive Technology
+
+Each menu item within a popover menu can use an aria-label that provides additional context that may not be required in a visible label. However, care must be taken to ensure that the visible label is included within the aria-label.
+
+For ease of use, actions can be assigned keyboard shortcuts and these are displayed on the menu item. This information is announced by a screen reader when each menu item is focussed.
+
+Toggle menu items allow the selection and deselection of something. Using aria-pressed allows the item's selection state to be announced by a screen reader when focussed.
+
+When moving between submenus in a popover menu, the nested submenu’s level is announced by a screen reader so that it is easy to understand whereabouts you are within the menu structure.
```

Comment:
```suggestion
When moving between sub-menus in a popover menu, the nested sub-menu’s level is announced by a screen reader so that it is easy to understand whereabouts you are within the menu structure.
```